docs/examples/federated_statistics_overview.rst

```diff
@@ -57,13 +57,13 @@ The main steps are:
     * provide server side configuration to specify target statistics and their configurations and output location
     * implement the local statistics generator (statistics_spec)
     * provide client side configuration to specify data input location
-    * The detailed example instructions can be found in `Data frame statistics <https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics/df_stats/README.md>`__
+    * The detailed example instructions can be found in `Data frame statistics <https://github.com/NVIDIA/NVFlare/tree/2.3/examples/federated_statistics/df_stats/README.md>`__
```

Comment:
I am wondering is there an easier way such as string substitution that could automatically substitute these links using correct version number

Reply:
For now, using the slashes around main to do a find replace does not result in any false positives, so this effectively functions as a string substitution, but the links with main actually do work too.